fury/actor/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""Actor module for FURY."""
```

Comment:
we try to follow the lazy loading technique.  

So, you need to create a `init.pyi` and this init should be updated

More information here: https://scientific-python.org/specs/spec-0001/